app/assets/stylesheets/dwp_checks.scss

```diff
@@ -26,6 +26,9 @@
   &.callout-part {
     background: $orange;
   }
+  &.evidence-check-reference {
```

Comment:
We should probably rename this file at some point, because the styles doesn't seem to have anything to do with dwp check. That's just a note.